src/PayU/PaymentsApi/PaymentsV4/Services/ResponseBuilder.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+
+namespace PayU\PaymentsApi\PaymentsV4\Services;
+
+use PayU\Alu\Response;
+use PayU\Alu\ResponseWireAccount;
+use PayU\PaymentsApi\PaymentsV4\Entities\Response\AuthorizationResponse;
+
+class ResponseBuilder
+{
+    /**
+     * @param AuthorizationResponse $authorizationResponse
+     * @return Response
+     */
+    public function buildResponse(AuthorizationResponse $authorizationResponse)
+    {
+        return $this->build($authorizationResponse);
```

Comment:
This buildResponse does not seem to do anything, could we call directly method build?

Reply:
done